src/index.js

```diff
@@ -0,0 +1,3 @@
+import { bra } from '../sw'
+
+document.querySelector('h1').textContent = bra('Sup from SW.js');
```

Comment:
Why is it called `bra`? I suspect this is trial code, but might make more sense to be called `greeting`?

Reply:
yeah, its just funny trial code.  "greeting" works too :D

Reply:
Though I generally try not to name a function the same as it's input for variable clarity.

Reply:
How about greetingEdit?